step-capstone/src/index.js

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './styles/index.css';
 import App from './App';
+import Trip from './components/Trip';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Trip />
```

Comment:
why is this no longer app?

Reply:
I put Trip here just for faster debugging. I'll put it back to App.

Reply:
This still needs to be fixed.